src/main/webapp/jsp/instructorFeedbackAjaxRemindParticularStudentsModal.jsp

```diff
@@ -2,14 +2,50 @@
 <%@ page import="teammates.common.util.Const"%>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 
-<c:forEach items="${data.responseStatus.noResponse}" var="userToRemindEmail">
-    <div class="checkbox">
-        <label>
-            <input type="checkbox" name="<%= Const.ParamsNames.SUBMISSION_REMIND_USERLIST %>" value="${userToRemindEmail}">
-            ${data.responseStatus.emailNameTable[userToRemindEmail]}
-        </label>
-    </div>
-</c:forEach>
+<table class="table-responsive table table-striped table-bordered">
+    <tr>
+        <th></th>
+        <th onclick="toggleSort(this);" class="button_sortsection button-sort-none">
+            Section
+            <span class="icon-sort unsorted"></span>
+        </th>
+        <th onclick="toggleSort(this);" class="button_sortteam button-sort-none">
+            Team
+            <span class="icon-sort unsorted"></span>
+        </th>
+        <th onclick="toggleSort(this);" class="button_sortname button-sort-none">
+            Student Name
+            <span class="icon-sort unsorted"></span>
+        </th>
+        <th onclick="toggleSort(this);" class="button_sortemail button-sort-none">
+            Email
+            <span class="icon-sort unsorted"></span>
+        </th>
+    </tr>
+    <c:forEach items="${data.responseStatus.noResponse}" var="userToRemindEmail">
+        <tr>
+            <td class="align-center">
+                <span class="checkbox">
```

Comment:
Can you confirm if it does anything?
The `checkbox` class on it should force it to `display: block`, so it should perform functionally the same as a `div`.